automated-tests/integration/gcp-gke/cluster.yml

```diff
@@ -21,3 +21,9 @@ spec:
     maxSize: 1
     volumeSize: 50
     instanceType: n1-standard-1
+  - name: my-spot-node-pool
+    minSize: 1
+    maxSize: 1
+    volumeSize: 50
+    spotInstance: true
```

Comment:
AFAIK, this attribute is wrong. You named it different on the `gke` installer

Reply:
made the variable name consistently `spot_instance`